katharsis-client/src/main/java/io/katharsis/client/internal/RelationshipRepositoryStubImpl.java

```diff
@@ -70,7 +70,7 @@ public void removeRelations(T source, Iterable<J> targetIds, String fieldName) {
 
 	private Serializable getSourceId(T source) {
 		ResourceField idField = sourceResourceInformation.getIdField();
-		return (Serializable) PropertyUtils.getProperty(source, idField.getUnderlyingName());
+		return (Serializable) idField.getAccessor().getValue(source);
```

Comment:
How about instead of `idField.getAccessor().getValue()` you just do `idField.getValue()` ? Less code for something often repeated